specification/apimanagement/control-plane/Microsoft.ApiManagement/preview/2017-03-01/apimportalsettings.json

```diff
@@ -0,0 +1,472 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApiManagementClient",
+    "description": "Use these REST APIs for performing operations on PortalSettings entity associated with your Azure API Management deployment. Using this entity you can manage settings for a Developer Portal.",
+    "version": "2017-03-01"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{apimBaseUrl}",
+    "useSchemePrefix": false,
+    "positionInOperation": "first",
+    "parameters": [
+      {
+        "name": "apimBaseUrl",
+        "description": "The management endpoint of the API Management service, for example https://myapimservice.management.azure-api.net.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "name": "Authorization",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "paths": {
+    "/portalsettings/signin": {
+      "get": {
+        "tags": [
+          "SignInSettings"
+        ],
+        "operationId": "SignInSettings_Get",
+        "description": "Get Sign-In settings.",
+        "x-ms-examples": {
+          "ApiManagementGetPortalSettingSignIn": {
+            "$ref": "./examples/ApiManagementGetPortalSettingSignIn.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Sign-In settings.",
+            "schema": {
+              "$ref": "#/definitions/PortalSigninSettings"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          }
+        }
+      },
+      "patch": {
```

Comment:
>patch [](start = 7, length = 5)

Is PATCH and PUT the only difference between `Update` and `CreateOrUpdate`?

Reply:
also If-Match is must in PATCH.. not in PUT.

---
In reply to: [163381931](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163381931) [](ancestors = 163381931)

Reply:
Same parameters in body are OK?

Reply:
yes.. actually it only has one parameter "enabled"

---
In reply to: [163391919](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163391919) [](ancestors = 163391919)